keyboards/cannonkeys/practice60/config.h

```diff
@@ -18,8 +18,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6464
+#define VENDOR_ID       0xCA04
+#define PRODUCT_ID      0x5060
```

Comment:
given `practice65` is 6565,
```suggestion
#define PRODUCT_ID      0x6060
```